### DISCUSSION TITLE
Ergast. What to expect

### DISCUSSION THREAD

**Discussion by kriebelkous** [theOehrly/Fast-F1#599] [Q&A] [answered]
Hi all, i am in the progress of replacing my website that relied on a local instance of the ergast api and database. 
I know that ergast is depricated, so it would not make sense to use a local docker of that again. But i have not found a good alternative yet. I have seen some hopefull news here about the ergast project being overhauled. That in itself is great news. I would be a shame to let all that information waste away on the web somewhere. But i cant seem to find any information on how it is going to be implemented. I am kind of at a crossroad in my development process and find that i just cant find the information to make the right call. The main question is, will i be able to run ergast (in any shape or form) locally after 2024?

Any info about this would be awesome.

**Comment by theOehrly** [comment, marked as answer]
Ergast will continue with a new name and new infrastructure, but with the same API. Further announcements will follow some time this summer, including a more publicly announced beta testing phase in the second half of the year. The server itself will be open source, so you should be able to run that locally. The new data ingestion process is still a work in progress and currently being worked on. Therefore, I cannot make any definite statement yet, how you'll be able to populate the local database. But I am confident that a local runtime will be possible without being too difficult to set up.

**Reply by kriebelkous** [reply]
That is very good news. Ill keep my eye out door the beta test and focus on other parts of my project for the time being. Thank you for the quick response and the work you (and your team) do here.

**Reply by walker-tx** [reply]
@theOehrly are you looking for contributors at this time?

**Reply by theOehrly** [reply]
@walker-tx in general, we're always happy about contributors. I've just posted a new announcement regarding the Ergast successor project here: https://github.com/theOehrly/Fast-F1/discussions/445#discussioncomment-10380182

Over the last few weeks, we were mainly stuck on problems that aren't easily solved by just having more people work on them. But you can have a look at the GitHub repo of the new project and if you see something that you want to work on or where you can offer help, you're always welcome.
